blackbox/docs/administration/ssl.txt

```diff
@@ -114,3 +114,272 @@ string. E.g.::
 For further information visit `psql documentation`_.
 
 .. _psql documentation: https://www.postgresql.org/docs/9.2/static/app-psql.html
+
+
+Setting up a Keystore/Truststore with a certificate chain
+=========================================================
+
+In case you need to setup a Keystore/Trustore, here are the commands
+to get you started. All the commands use a validity of 36500 days
+(about 100 years). You might want to use less.
+
+.. NOTE::
+
+   Only follow the optional steps if you want to create your own
+   Certificate Authority. Otherwise, please request a signed certificate
+   from one of the CAs bundled with Java.
+
+
```

Comment:
I find it easier to read the source with two lines between the sections. It doesn't affect the output. If you don't mind, I'll keep it.